javascript-arrays/main.js

```diff
@@ -0,0 +1,38 @@
+const colors = ['red', 'white', 'blue'];
+
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+console.log(
+  'America is ' +
```

Comment:
I would probably write this all on one line for readability. Not a requirement though :)